### ISSUE TITLE
Additional Window not loading properly

### ISSUE THREAD

**Issue by JayZeeeS** [fyne-io/fyne#3653]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

When opening more than one window with the Window.Show() method after resizing with Window.Resize(), the content in said window is not visible, while the main window is still fine. See the Canvas and text windows in the screenshots. When resized manually

### How to reproduce

Run the provided code.

### Screenshots

![image](https://user-images.githubusercontent.com/107275650/218180795-3d224869-c47c-44c3-9b88-5b154c8f7574.png)
![image](https://user-images.githubusercontent.com/107275650/218180823-610b6fe3-8151-40fc-acd2-3fe676dfdc2b.png)
![image](https://user-images.githubusercontent.com/107275650/218180850-94f1b0c5-ecb7-4da1-ba4f-23524a1da644.png)

### Example code

```
go
package main

import (
	"fyne.io/fyne/v2"
	"fyne.io/fyne/v2/app"
	"fyne.io/fyne/v2/widget"
)

func main() {
	a := app.New()
	w := a.NewWindow("window1")
	w.SetContent(widget.NewLabel("Hello"))

	w2 := a.NewWindow("window2")
	w2.SetContent(widget.NewLabel("World"))

	w.Resize(fyne.NewSize(300, 300))
	w2.Resize(fyne.NewSize(300, 300))
	w.Show()
	w2.Show()
	a.Run()
}
```

### Fyne version

2.3.0

### Go compiler version

1.20

### Operating system

Linux

### Operating system version

WSL Ubuntu-20.04

**Comment by Jacalz** [comment]
I don’t think we officially support WSL. It behaves very strangely in some cases. To be honest, I don’t know if this is a bug in Fyne or in the emulation that WSL is doing.

**Comment by Bluebugs** [comment]
It is my understanding that WSL is actually a RDP over a Wayland session.

**Comment by andydotxyz** [comment]
I'm not sure what to do here. Is anyone able to work on WSL2?
I don't think it's on the officially supported list.
Should it be?

**Comment by Bluebugs** [comment]
I got application to display last time I tried, but they had the short coming of Wayland application as that's what's running under the hood.

**Comment by Jacalz** [comment]
> It is my understanding that WSL is actually a RDP over a Wayland session.

I'm not entirely sure to which degree that it is the case.  I know they have been working on translating OpenGL and Vulkan into DirectX12 among other things. They should probably also have some XWayland implementation as well. It seems hard to target WSL in development and doing specific fixes for it as it is running Linux binaries and then doing some kind of translation. We will always be at the mercy of the Linux version together with any possible issues in the translation.

> I don't think it's on the officially supported list. Should it be?

I know that quite a few people would like it to work but I don't see how we can change things on our end to make it work any better.

**Comment by Bluebugs** [comment]
The translation to windows driver is orthogonal to the protocol used to manage windows. It is opengl and as long as we follow opengl, that should just work. I have not seen any rendering issue when I tried.

There was not any X server exposed when I tried, but just Wayland last time. They may have added it since then. Would be worth checking today. In any case of it is XWayland or Wayland, it behave the same as on Linux. It is literally running Wayland with a rdp backend in a vm that run Linux.

**Comment by Jacalz** [comment]
There might also be differences between WSL and WSL2. I don't know if both still are available. They might be.

**Comment by Jacalz** [comment]
You're right. There might not be X11 support by default. Perhaps it just is bugs in our Wayland code that are surfacing (it is far from complete at the moment)?

**Comment by andydotxyz** [comment]
I'm surprised that this just works then - as our wayland code is odd by default...

**Comment by andydotxyz** [comment]
It may relate to the Rosetta discussion as these are both emulations on an OS that we support natively - so should we continue to put work into supporting the emulation? https://github.com/fyne-io/fyne/issues/3971
